l1-contracts/test/Decoder.t.sol

```diff
@@ -165,13 +167,13 @@ contract DecoderTest is Test {
       hex"0000002400000008aafdc7aa93e78a700000001497aee30906a86173c86c6d3f108eefc36e7fb014";
     (bytes32 logsHash, uint256 bytesAdvanced) = helper.computeKernelLogsHash(emptyKernelData);
 
-    // Note: First 32 bytes occupied by logs hash of previous iteration, the rest occupied by I2_LOGS
+    // Note: First 32 bytes occupied by logs hash of previous iteration , the rest occupied by I2_LOGS
     bytes32 referenceLogsHash = bytes32(
```

Comment:
Same thing down here, the library should be used for dealing with it nicely. 

Also, is there a reason this value is hadcoded and not using `referenceLogsHash`?

Reply:
Addressed in [90d4bb1](https://github.com/AztecProtocol/aztec-packages/pull/615/commits/90d4bb1f72a7894ab04518a1565132d7e29ab923).

My original intention was to hardcode value to show that it's the hash from the previous iteration. But I refactored it and I think it's better as it is now.